PixivModelFanbox.py

```diff
@@ -193,12 +179,22 @@ def parseBody(self, jsPost):
                                      self.body_text,
                                      jsPost["body"]["imageMap"][imageId]["originalUrl"],
                                      jsPost["body"]["imageMap"][imageId]["thumbnailUrl"])
+                    for image in jsPost["body"]["imageMap"]:
```

Comment:
rather than iterating the dictionary, you can append the image using the `imageId` directly.
```suggestion
                    self.images.append(jsPost["body"]["imageMap"][imageId]["originalUrl"])
                    self.embeddedFiles.append(jsPost["body"]["imageMap"][imageId]["originalUrl"])
```